### ISSUE TITLE
The longest Journey needs ResidualVM 0.4.0git

### ISSUE THREAD

**Issue by Brunobasta** [haikuports/haikuports#5165]
I just run The longest Journey with ResidualVM on Windows which uses the unstable build and it works great.
I could not find a way to run with Haiku and ResidualVM 0.3.1-7
I also could not find where to get the ResidualVM 0.4.0git  version...
Maybe you have a clue (glue?)
It is a great game and I like to test it...
It is advertised big on the ResidualVM Homepage!
Thanks, now I try to run Escape from Moneky Island... which schould work with Ver. 0.3.1-7 (yes it is running fine with 60fps!)

**Comment by Brunobasta** [comment]
https://imgur.com/JC0YAQg shows how it should work...

**Comment by Brunobasta** [comment]
Works fine here, should be closed and merged to HaikuDepot for 64bit...
Thanks again Gerasim for the porting.

**Comment by Begasus** [comment]
@Brunobasta does it work on the 0.3.1 version we have atm?From their website they announce you should use a daily build for it?

**Comment by Brunobasta** [comment]
No, it does not work with the 0.3.1 version! It works only withe the newest daily build 0.4.0git I got from 3Deyes.

**Comment by Begasus** [comment]
Should be fixed in https://github.com/haikuports/haikuports/commit/7b84fa8dba7d88c74ecdc502b818a4bfae1e82fa
@Brunobasta could you check?

**Comment by Brunobasta** [comment]
Yes the 64bit version works nice now.
The 32bit version will crash when try to load a game.

**Comment by Begasus** [comment]
New release for ScummVM is on the way :)

**Comment by Brunobasta** [comment]
Not sure if it is a good idea to bundle SCUMMVM together with ResidualVM.
ResidualVM is very slow on Haiku... Scummvm is much faster especially the GUI.
Let's see what happens...

**Comment by Begasus** [comment]
So far I have been able to build the latest version, but it doesn't launch :/ (need to see if disabling all engines does make it launch)
On the part of the merge, it's something that's been decided in the ScummVM team (not every one was happy about it too, I even think one of the people from ResidualVM stepped out of the project because of it)
